support.md

```diff
@@ -0,0 +1,29 @@
+Support
```

Comment:
Can we please change the casing of this file to be `SUPPORT.md` similar to the other md files in the repo.

Reply:
Done